### ISSUE TITLE
Feature request: BigQuery Implementation

### ISSUE THREAD

**Issue by jjjermiah** [Cyclenerd/google-cloud-pricing-cost-calculator#102]
**Is your feature request related to a problem? Please describe.**
I believe incorporating a cost estimation for BigQuery would be great is all. 

I believe this is on the to-do list, but if work hasn't started, I would love to contribute to an implementation for it. 
If some thought has gone into this, I can try to work off that as well?

What are the usual requirements for including a GCP product that is new to this project?

**Comment by Cyclenerd** [comment]
You are right, it is on the TODO list but not started. The BigQuery price structure is imo. somewhat complicated. You have the storage space and the querys that are based on the amount of data. This is not easy to estimate. In addition you also have the possibility of commitment aka slots (<https://cloud.google.com/bigquery/docs/slots>). All this has scared me away from tackling the topic so far. So far you seem to be the only one who misses BigQuery ;)

In this [README](https://github.com/Cyclenerd/google-cloud-pricing-cost-calculator/tree/master/build) I describe how the costs are calculated. Based on this, your individual costs will be calculated. Pull Request welcome.
